src/main/java/frc/robot/library/controlsystem/PwrDistribution.java

```diff
@@ -5,18 +5,18 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-package frc.robot.Library.ControlSystem;
+package frc.robot.library.controlsystem;
 
 import edu.wpi.first.wpilibj.PowerDistribution;
 import edu.wpi.first.wpilibj.PowerDistribution.ModuleType;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.CAN_IDs;
 
-public class PwrDistributionSubSys extends SubsystemBase {
+public class PwrDistribution extends SubsystemBase {
```

Comment:
Shouldn't be a SubSystem as it doesn't use any of its methods. This will take up time in our command scheduler loop.